src/IssueViz.Security.UnitTests/Hotspots/HotspotMatcherTests.cs

```diff
@@ -30,6 +36,188 @@ public class HotspotMatcherTests
     [TestMethod]
     public void MefCtor_CheckExports()
     {
-        MefTestHelpers.CheckTypeCanBeImported<NoOpHotspotMatcher, IHotspotMatcher>();
+        MefTestHelpers.CheckTypeCanBeImported<HotspotMatcher, IHotspotMatcher>();
+    }
+    
+    [TestMethod]
+    public void IsMatch_NoHashDifferentLineSameMessage_ReturnsTrue()
+    {
+        const string ruleId = "rule1";
+        const string filePath = "A:\\ny\\p\\ath";
+        const string serverPath = "p\\ath";
+        const string message = "message";
+        var testSubject = CreateTestSubject();
+
+        testSubject.IsMatch(CreateLocalHotspot(ruleId, message, "abchash", filePath, 10),
+                CreateServerHotspot(ruleId, message, null, serverPath, 20))
+            .Should()
+            .BeTrue();
+    }
+    
+    [TestMethod]
+    public void IsMatch_NoHashSameLine_ReturnsTrue()
+    {
+        const string ruleId = "rule1";
+        const string filePath = "A:\\ny\\p\\ath";
+        const string serverPath = "p\\ath";
+        var testSubject = CreateTestSubject();
+
+        testSubject.IsMatch(CreateLocalHotspot(ruleId, "message1", "abchash", filePath, 10),
+                CreateServerHotspot(ruleId, "message2", null, serverPath, 10))
+            .Should()
+            .BeTrue();
+    }
+    
+    [TestMethod]
+    public void IsMatch_SameIssueDifferentHash_ReturnsTrue()
```

Comment:
A number of the tests have "SameIssue" in the name; it's not clear what that means, or what the test is checking.